save-common/src/jvmMain/kotlin/org/cqfn/save/entities/Agent.kt

```diff
@@ -10,11 +10,9 @@ import javax.persistence.ManyToOne
  */
 @Entity
 class Agent(
-
     var containerId: String,
 
     @ManyToOne
     @JoinColumn(name = "execution_id")
-    var execution: Execution,
-
+    var execution: Execution?,
```

Comment:
Why is execution nullable? In DB it has nullable = false and FK

Reply:
That's because when orchestrator updates agent statuses based on heartbeats, it doesn't know `Execution` for the agent. It actually needs to save an `AgentStatus`, finding corresponding `agent.agent_id` based on `agent.container_id`.

Reply:
Created a more correct method for this

Reply:
Also, made `Execution` lazily fetched, because on agent status update we only need agent id from this entity